specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/AppConfiguration.json

```diff
@@ -50,6 +50,58 @@
           "type": "string"
         }
       }
+    },
+    "AppConfigurationKSnapshotCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AppConfiguration.SnapshotCreated event.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of snapshot that was created.",
+          "type": "string"
+        },
+        "etag": {
+          "description": "The etag representing the new state of the snapshot.",
+          "type": "string"
+        },
+        "syncToken": {
+          "description": "The sync token representing the server state after the event.",
+          "type": "string"
+        }
+      },
+      "x-ms-examples": {
+        "snapshotCreatedCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/snapshot_created.json"
+        },
+        "snapshotCreatedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/snapshot_created.json"
+        }
+      }
+    },
+    "AppConfigurationKSnapshotModifiedEventData": {
```

Comment:
Is the K a typo?

Reply:
yes, K is a leftover from previous name which we decided not to use. K is removed in both event name